pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go

```diff
@@ -641,6 +641,109 @@ func TestCSILimits(t *testing.T) {
 	}
 }
 
+func TestCSILimitsQHint(t *testing.T) {
```

Comment:
Can you add a test case to verify that the function just ignores deleted Pod's PVC(s) that are already deleted and evaluates only living PVC(s)?

Reply:
Sure. Will this change satify you?
https://github.com/kubernetes/kubernetes/pull/121508/commits/0f86bd2401024d371bc2f3b677a40c1efcb7f348